### PR TITLE
Task/Admin_Login_Page

### PR DESCRIPTION
**Pull request by CaneteJoash**
## Related Links

- Task Link: https://app.asana.com/0/1202252431396579/1202434821847342
- Route Link: http://localhost:3000/admin/login

## Description

- Task 1:  Create [Admin] [Markup] Login Page 
## Notes
N/A

## Screenshots
image

### CODE REVIEW

File: src/pages/admin/LoginPage/index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import InputField from '../../components/inputField';
+
+const AdminLoginPage = () => {
+  const handleSubmit = (e) => {
+    e.preventDefault(e.target.value);
+    window.location = '/admin/home';
+  };
+
+  const handleOnChange = (e) => {
+    console.log(e.target.value);
```
Comment (ErickDoroy) [inline src/pages/admin/LoginPage/index.js]: If not use please comment out or just erase the 'console.log()' and 'e'

File: src/pages/components/inputField/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const InputField = ({ name, type, placeholder, onChange }) => (
+  <div className='mt-[30px] flex flex-col'>
+    <div>
+      <span className='text-base font-bold text-brown-900'>{name}</span>
```
Comment (ErickDoroy) [inline src/pages/components/inputField/index.js]: use `<p>` element to give margin bottom between the input field and input title

**Review by ume45876139 — APPROVED** [review]
# Working

<details>
<summary>Browser</summary>

Chrome

スクリーンショット 2022-06-15 16 44 13

Safari

スクリーンショット 2022-06-15 17 14 31

Opera

スクリーンショット 2022-06-15 17 19 43

Firefox

スクリーンショット 2022-06-15 17 20 36

</details>

- Markup is correct
- No deformation at each browser

# Testing data
test device: Chrome, Safari, Opera, Firefox
test time: 0.75h
comment: 0
iteration: 0
total test time: 0.75

**Review by ErickDoroy — APPROVED** [review]

File: src/pages/admin/LoginPage/index.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  const handleSubmit = (e) => {
+    e.preventDefault(e.target.value);
+    window.location = '/admin/home';
+  };
+
+  const handleOnChange = (e) => e.target.value;
+
+  return (
+    <div className='flex h-screen items-center justify-center bg-gradient-to-b from-orange-900'>
+      <div className='pb:0 w-[300px] rounded-[5px] bg-sky-100 p-7 sm:w-[545px] sm:p-14 sm:pb-5'>
```
Comment (egiegarciano14) [inline src/pages/admin/LoginPage/index.js]:
```suggestion
      <div className='pb:0 w-[300px] rounded-[5px] bg-orange-100 p-7 sm:w-[545px] sm:p-14 sm:pb-5'>
```

**Comment by egiegarciano14** [comment]
Pwede nimo himo kag folder named 'shared' sud sa src folder nya e butang didto ang Button and InputField :bow: 
image

**Review by egiegarciano14 — APPROVED** [review]

**Pull request merged**